### DISCUSSION TITLE
Boomerang [US]

### DISCUSSION THREAD

**Discussion by SkylerDaKat** [iptv-org/community#926] [Q&A]
### Channel ID

Boomerang.us

### Notes (optional)

self explanatory 

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19857
